bedrock/products/templates/products/vpn/landing.html

```diff
@@ -192,7 +192,7 @@ <h3>{{ ftl('vpn-landing-faq-compatibility-question-heading') }}</h3>
       </summary>
       <p class="vpn-faq-item-detail">{{ ftl('vpn-landing-faq-compatibility-question-desc') }}</p>
 
-      <ol class="vpn-faq-item-detail mzp-u-list-styled">
+      <ul class="vpn-faq-item-detail mzp-u-list-styled">
```

Comment:
Need to change the closing tag to `</ul>` as well.

Reply:
AH! Reviewer fail. Thanks Craig.

Reply:
D'oh, I commented just as it was merged. This shouldn't cause any visible issues, most browsers will be able to handle it. Just a minor fix we can do for the next time the page changes.

Reply:
Fixed in #10029